android/src/main/java/com/pspdfkit/flutter/pspdfkit/FlutterPdfActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.pspdfkit.flutter.pspdfkit;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+
+import com.pspdfkit.document.PdfDocument;
+import com.pspdfkit.ui.PdfActivity;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import io.flutter.plugin.common.MethodChannel.Result;
+
+/**
+ * For communication with the PSPDFKit plugin, we keep a static reference to the current
+ * activity.
+ */
+public class FlutterPdfActivity extends PdfActivity {
+
+    private static FlutterPdfActivity currentActivity;
+    private static AtomicReference<Result> loadedDocumentResult = new AtomicReference<>();
+
+    public static void setLoadedDocumentResult(Result result) {
+        loadedDocumentResult.set(result);
+    }
+
+    @Override
+    protected void onCreate(Bundle bundle) {
+        super.onCreate(bundle);
+        bindActivity();
```

Comment:
You should reset this in `onDestroy` otherwise you leak the activity when the user closes it.